### PR TITLE
fixing dropdowns

### PR DESCRIPTION
**Pull request by puneetg1983**

### CODE REVIEW

File: AngularApp/projects/diagnostic-data/src/lib/components/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export class DropdownComponent extends DataRenderBaseComponent {
       const row = table.rows[i];
       this.label = row[labelColumn];
       const key: string = row[keyColumn];
-      const selected: boolean = row[selectedColumn].toLowerCase() === 'true';
+      const selected: boolean = String(row[selectedColumn]).toLowerCase() === 'true';
```
Comment (stevenernst131) [inline AngularApp/projects/diagnostic-data/src/lib/components/dropdown/dropdown.component.ts]: isn't row[selectedColumn] already a boolean? may just be able to get rid of the translating it to string and doing the comparison?

Comment (puneetg1983) [reply AngularApp/projects/diagnostic-data/src/lib/components/dropdown/dropdown.component.ts]: It is currently a boolean but ideally the backend should send a string and I was thinking that will get fixed at some point ? So if it changes in future, we dont have to change this part....

Comment (stevenernst131) [reply AngularApp/projects/diagnostic-data/src/lib/components/dropdown/dropdown.component.ts]: No the back end will not change, I don't think.

Comment (ShekharGupta1988) [reply AngularApp/projects/diagnostic-data/src/lib/components/dropdown/dropdown.component.ts]: If the changes are deployed everywhere (meaning all georegions), then just using the boolean value const selected: boolean = row[selectedCloumn] should be fine..

However, the current change looks more guarded against any future changes that might happen.

**Comment by stevenernst131** [comment]
I checked in fix for this

**Pull request closed without merging**